lib/crowdflower/job.rb

```diff
@@ -28,8 +28,8 @@ def self.upload(file, content_type, job = nil, opts = {})
       job_uri = job ? "/#{job.kind_of?(Job) ? job.id : job}" : ""
       conn = job.kind_of?(Job) ? job.connection : self.connection
       upload_uri = "#{resource_uri}/#{job_uri}/upload".squeeze("/")
-      res = conn.post(upload_uri, 
-        :body => File.read(file), 
+      res = conn.post(upload_uri,
+        :body => File.read(file),
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.